### ISSUE TITLE
locale in KDE

### ISSUE THREAD

**Issue by geckolinux** [SpiralLinux/SpiralLinux-project#133]
### Discussed in https://github.com/orgs/SpiralLinux/discussions/132

<div type='discussions-op-text'>

<sup>Originally posted by **waternumbers** September 28, 2022</sup>
Hi

Liking the KDE version of SpiralLinux but...

Installing from the live media I select settings for the UK: en-gb language, keyboard layout and London time zone.
On installation only the time zone is correct, keyboard and language are set to en-us.
Also the en-gb language packs for firefox, hunspell and libreoffice are not installed (a wireless connection was available in the live session)

Given the language support link on desktop is this a bug or expected behaviour?

If it is expected behaviour it is easy to install the language packs and alter the system settings for one user - but what is the best way of altering the settings for multiple users?

Thanks

Paul</div>

**Comment by deknos** [comment]
In general setting the language support to a language and the layout is.. hard? my language and timezone is german, and it was surprisingly hard. It "felt" harder than with a normal debian install to configure for the desktop and system environment all the locales and set them to german?

**Comment by deknos** [comment]
uh, sorry, i did not see your answer, i will do a second installation tomorrow just to be sure.

**Comment by geckolinux** [comment]
Thanks!

**Comment by deknos** [comment]
Hey, i finally came around doing an installation.

i fetched SpiralLinux_Plasma_11.220925_x86-64.iso changed to german in the live environment via set system language and started the installer. the installer saw that i wanted german and accepted that. after installation and reboot:

* most of kde is english, though the name of the standard system network connection (ethernet) is german
* firefox is english
* there's no LC_* or LANG variable.

what do i do wrong? should i not have chosen german as system language in the live environment?

**Comment by geckolinux** [comment]
@deknos Hi there, sorry for the trouble, please try this:
https://github.com/orgs/SpiralLinux/discussions/132#discussioncomment-3757509

**Comment by deknos** [comment]
This works almost for all. LC_ALL is not set but quite a lot of other LC_ variables and LANG is set.

Now, this is just a minor issue, and i do not know if this upstream is not also a problem, but the message when KDE activates the network connection is in part STILL english. if you want to  i can add a screenshot :)

**Comment by deknos** [comment]
also, the right click menu of kde is still english, even after reboot.

**Comment by deknos** [comment]
also the env variable LANGUAGE is set to de:en_US and the "task-german-desktop" and "task-german-kde-desktop" tasks are not installed. perhaps this will help, but i think i have to remove all configs

**Comment by deknos** [comment]
Well, with installing locales-all, installing task-german-desktop and task-german-kde-desktop and removing all Dot-Files/Dot-Directories and Reboot, it worked, everything is german :)

**Comment by geckolinux** [comment]
Glad you figured it out, thanks for reporting what you did.

> removing all Dot-Files/Dot-Directories

That's a rather extreme solution. The dot files mentioned [here](https://github.com/orgs/SpiralLinux/discussions/132#discussioncomment-3757509) should be enough.

**Comment by deknos** [comment]
> That's a rather extreme solution. 

Yes, it is. I just wanted to make sure this works. I will now reinstall and try to get this done on the first try. if you want, i can document that here. because i think if i install the tasks, most of that is configured automatically.

**Comment by geckolinux** [comment]
Sure, please let me know. Appreciate the testing.
